eeps/eep-0053.md

```diff
@@ -1,5 +1,5 @@
     Author: Rickard Green <rickard(at)erlang(dot)org>
-    Status: Accepted/24.0 Proposal is to be implemented in OTP release 24
+    Status: Final/24.0 Implemented in OTP release 24.0
     Type: Standards Track
     Erlang-Version: 24.0
     Created: 01-Sept-2019
```

Comment:
I'm not able to suggest the Post-History here.
```
    Post-History: 06-Sep-2023
```

Reply:
Same here.  In most cases there is no preceding discussion - the EEP is often used to start the discussion, and the discussion often takes place around the code change, e.g the GitHub PR